### ISSUE TITLE
Would you inform about the wide-shallow network?

### ISSUE THREAD

**Issue by amoeba04** [swz30/Restormer#44]
Hello,

In the ablation study, you compared deeper vs wider Restormer.
I'm wondering about the wider Restormer you mentioned, so could you inform me of the details of it?

**Comment by swz30** [comment]
Hi @amoeba04 

Here are the network parameters:

**For Dim 64:**

```
class Restormer(nn.Module):
    def __init__(self, 
        inp_channels=3, 
        out_channels=3, 
        dim = 64,
        num_blocks = [2,3,3,4], 
        num_refinement_blocks = 3,
        heads = [1,2,4,8],
        ffn_expansion_factor = 2.53,
        bias = False,
        LayerNorm_type = 'WithBias',   ## Other option 'BiasFree'
        dual_pixel_task = False
    ):
```

**And for Dim 80:**

```
class Restormer(nn.Module):
    def __init__(self, 
        inp_channels=3, 
        out_channels=3, 
        dim = 80,
        num_blocks = [1,2,2,2], 
        num_refinement_blocks = 3,
        heads = [1,2,4,8],
        ffn_expansion_factor = 2,
        bias = False,
        LayerNorm_type = 'WithBias',   ## Other option 'BiasFree'
        dual_pixel_task = False
    ):
```

**Comment by amoeba04** [comment]
Oh, it's clear. Thank you!
